hs_core/models.py

```diff
@@ -1465,10 +1465,18 @@ def delete(self, using=None):
                 if fl.fed_resource_file_name_or_path.find(self.short_id) >= 0:
```

Comment:
@pkdash I was thinking of something else when giving the comments above. Now that I looked at the code again, I think you are right, I can simplify the logic using root_path. Will do so and make sure all tests pass before asking your final review again. Thanks.